homeassistant/components/sensor/awair.py

```diff
@@ -148,57 +162,53 @@ def state(self):
         return self._data.data[self._type]
 
     @property
-    def score(self):
-        """Return the aggregate Awair score."""
-        return self._data.data[ATTR_SCORE]
-
-    @property
-    def timestamp(self):
-        """Return the timestamp of the last sensor reading."""
-        return self._data.data[ATTR_TIMESTAMP]
+    def device_state_attributes(self):
+        """Return additional attributes."""
+        return self._data.attrs
 
     @property
     def unique_id(self):
         """Return the unique id of this entity."""
         return "{}_{}".format(self._uuid, self._type)
 
-    @property
-    def uuid(self):
-        """Return the API UUID of this entity."""
-        return self._uuid
-
     @property
     def unit_of_measurement(self):
         """Return the unit of measurement of this entity."""
         return self._unit_of_measurement
 
+    @property
+    def should_poll(self):
+        """HA should poll us for state."""
+        return True
```

Comment:
This is the default

Reply:
Thanks - that wasn't immediately obvious from the code. I'll remove it.